samples/src/Microsoft.Azure.IIoT.App/src/Pages/Discoverers.razor

```diff
@@ -49,6 +49,7 @@
     <thead>
         <tr>
             <th class="width-large">Discovery Module Id</th>
+            <th class="width-medium">Site</th>
```

Comment:
This should be called Gateway Id.   But if the idea is to show the site, then below should refer to the SiteId value in the API model.